simple/stats/events_importer.py

```diff
@@ -259,3 +301,14 @@ def _write_debug_csvs(self) -> None:
                    self.debug_resolve_fh)
       self.debug_resolve_fh.write_string(
           self.debug_resolve_df.to_csv(index=False))
+
+
+# Utility methods
+def _time_period(date_str: str, period: TimePeriod) -> str:
+  date = date_parse(date_str)
+  if period == TimePeriod.DAY:
+    return date.strftime("%Y-%m-%d")
+  if period == TimePeriod.YEAR:
+    return date.strftime("%Y")
+  # Default to month
+  return date.strftime("%Y-%m")
```

Comment:
maybe default to year since we mostly have yearly data?

Reply:
Done.